account_payment_partner/tests/test_account_payment_partner.py

```diff
@@ -0,0 +1,588 @@
+            {
+                "name": "J2",
+                "code": "J2",
+                "type": "bank",
+                "company_id": cls.company_2.id,
+                "bank_acc_number": "552344",
+            }
+        )
+
+        cls.supplier_payment_mode = cls.payment_mode_model.create(
+            {
+                "name": "Suppliers Bank 1",
+                "bank_account_link": "variable",
+                "payment_method_id": cls.manual_out.id,
+                "show_bank_account_from_journal": True,
+                "company_id": cls.company.id,
+                "fixed_journal_id": cls.journal_c1.id,
+                "variable_journal_ids": [(6, 0, [cls.journal_c1.id])],
+            }
+        )
+
+        cls.supplier_payment_mode_c2 = cls.payment_mode_model.create(
+            {
+                "name": "Suppliers Bank 2",
+                "bank_account_link": "variable",
+                "payment_method_id": cls.manual_out.id,
+                "company_id": cls.company_2.id,
+                "fixed_journal_id": cls.journal_c2.id,
+                "variable_journal_ids": [(6, 0, [cls.journal_c2.id])],
+            }
+        )
+
+        cls.customer_payment_mode = cls.payment_mode_model.create(
+            {
+                "name": "Customers to Bank 1",
+                "bank_account_link": "variable",
+                "payment_method_id": cls.manual_in.id,
+                "company_id": cls.company.id,
+                "fixed_journal_id": cls.journal_c1.id,
+                "refund_payment_mode_id": cls.supplier_payment_mode.id,
+                "variable_journal_ids": [(6, 0, [cls.journal_c1.id])],
+            }
+        )
+        cls.supplier_payment_mode.write(
+            {"refund_payment_mode_id": cls.customer_payment_mode.id}
+        )
+
+        cls.customer = (
+            cls.env["res.partner"]
+            .with_company(cls.company.id)
+            .create(
+                {
+                    "name": "Test customer",
+                    "customer_payment_mode_id": cls.customer_payment_mode,
+                }
+            )
+        )
+
+        cls.supplier = (
+            cls.env["res.partner"]
+            .with_company(cls.company.id)
+            .create(
+                {
+                    "name": "Test supplier",
+                    "supplier_payment_mode_id": cls.supplier_payment_mode,
+                }
+            )
+        )
+        cls.supplier_bank = cls.env["res.partner.bank"].create(
+            {
+                "acc_number": "5345345",
+                "partner_id": cls.supplier.id,
+            }
+        )
+        cls.supplier.with_company(
+            cls.company_2.id
+        ).supplier_payment_mode_id = cls.supplier_payment_mode_c2
+
+        cls.invoice_account = cls.env["account.account"].search(
+            [
+                ("account_type", "=", "liability_payable"),
+                ("company_id", "=", cls.company.id),
```

Comment:
```suggestion
                ("company_ids", "in", cls.company.id),
```
The model `account.account` now has `company_ids` instead of `company_id`